src/utilities/filetypes/EpwFile.hpp

```diff
@@ -843,6 +862,120 @@ class UTILITIES_API EpwDesignCondition
   double m_extremeN50YearsMaxDryBulb;
 };
 
+/** EpwGroundTemperatureDepth is one line from the EPW file. All floating point numbers are stored as strings,
+* but are checked as numbers.
+*/
+class UTILITIES_API EpwGroundTemperatureDepth
+{
+ public:
+  /** Create an empty EpwGroundTemperatureDepth object */
+  EpwGroundTemperatureDepth();
+  /** Create an EpwGroundTemperatureDepth object with specified properties */
+  EpwGroundTemperatureDepth(double groundTemperatureDepth, double soilConductivity, double soilDensity, double soilSpecificHeat, double jan,
+                            double feb, double mar, double apr, double may, double jun, double jul, double aug, double sep, double oct, double nov,
+                            double dec);
+  // Static
+  /** Returns the units of the named field */
+  static boost::optional<std::string> getUnitsByName(const std::string& name);
+  /** Returns the units of the field specified by enumeration value */
+  static std::string getUnits(EpwDepthField field);
+  // Data retrieval
+  /** Returns the double value of the named field if possible */
+  boost::optional<double> getFieldByName(const std::string& name);
+  /** Returns the dobule value of the field specified by enumeration value */
+  boost::optional<double> getField(EpwDepthField id);
+  // Conversion
+  /** Create an EpwGroundTemperatureDepth from an EPW-formatted string */
+  static boost::optional<EpwGroundTemperatureDepth> fromGroundTemperatureDepthsString(const std::string& line);
+  /** Create an EpwGroundTemperatureDepth from a list of EPW depths as strings */
+  static boost::optional<EpwGroundTemperatureDepth> fromGroundTemperatureDepthsString(const std::vector<std::string>& list);
```

Comment:
```suggestion
  static boost::optional<EpwGroundTemperatureDepth> fromGroundTemperatureDepthsStrings(const std::vector<std::string>& list);
```

I would refrain from using "list", to be confused by std::list